vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -296,8 +316,9 @@ def submit_metrics_callback(self, query_results):
                 if not valid_values:
                     self.log.debug(
                         "Skipping metric %s because the value returned by vCenter"
-                        " is negative (i.e. the metric is not yet available).",
+                        " is negative (i.e. the metric is not yet available). values: %s",
                         to_string(metric_name),
+                        list(result.value),
```

Comment:
Example:

```
2020-09-15 11:22:51 UTC | CORE | DEBUG | (pkg/collector/python/datadog_agent.go:123 in LogMessage) | vsphere:9ee463a9fc5cb748 | (vsphere.py:317) | Skipping metric datastore.throughput.contention.avg because the value returned by vCenter is negative (i.e. the metric is not yet available). values: [-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1]
```